### PR TITLE
Add a note about Skia build issues on Windows

### PR DESCRIPTION
**Pull request by tronical**

### CODE REVIEW

File: docs/reference/src/advanced/backends_and_renderers.md [hunk 1]
```diff
@@ -80,3 +80,20 @@ The Qt renderer comes with the [Qt backend](backend_qt.md) and renders using QPa
...
+#### Troubleshooting
+
+You may run into compile issues when enabling the Skia renderer. The following sections track
+issues we're aware of and how to resolve them.
+
+* Compilation error on Windows with messages about multiple source files and unused linker input
+
+  You may see compile errors that contain this error and warning from clang-cl:
+  ```
+   clang-cl: error: cannot specify '/Foobj/src/fonts/fontmgr_win.SkFontMgr_indirect.obj' whencompiling multiple source files
```
Comment (ogoffart) [inline docs/reference/src/advanced/backends_and_renderers.md:92]: > whencompiling

Is the space missing in the original error message?

File: docs/reference/src/advanced/backends_and_renderers.md [hunk 2]
```diff
@@ -80,3 +80,31 @@ The Qt renderer comes with the [Qt backend](backend_qt.md) and renders using QPa
...
+   clang-cl: warning: Hausmann/.cargo/registry/src/index.crates.io-6f17d22bba15001f/skia-bindings-0.66.0/skia: 'linker' input unused [-Wunused-command-line-argument]
```
Comment (ogoffart) [inline docs/reference/src/advanced/backends_and_renderers.md:93]: Your name is "leaking"
If you don't mind, that's fine

**Review by ogoffart — APPROVED** [review]

File: docs/reference/src/advanced/backends_and_renderers.md (hunk 2 above)
Comment (tronical) [reply docs/reference/src/advanced/backends_and_renderers.md:93]: I don't mind :)

File: docs/reference/src/advanced/backends_and_renderers.md (hunk 1 above)
Comment (tronical) [reply docs/reference/src/advanced/backends_and_renderers.md:92]: Ah no, it's in the original message. I'll add it. Well spotted.

**Pull request merged**